### ISSUE TITLE
Test stacking time steps into channel dimension

### ISSUE THREAD

**Issue by DWGodwin** [ClarkCGA/gfm-biomassters#2]
Per the research meeting today, my next step is to input all data into Prithvi, stacking the time dimension into the channel dimension, resulting in 36 channels and 4 time steps.

**Comment by DWGodwin** [comment]
Changes need to be made to the dataloader for this step:
- The shape of the outputs are correct (36, 4, 224, 224), but I should write a function to plot each band of an example to ensure that the bands are stacked correctly.
- The pipeline runs until this error currently: ```The size of tensor a (36) must match the size of tensor b (12) at non-singleton dimension```. In order to fix this, the means and standard deviations should be repeated in the config. This makes it crucial that the bands are stacked in the expected order.

**Comment by DWGodwin** [comment]
Experiments show slightly worse performance than 4-time-step during training and validation. However, I think that may be due to the 4 time step script eliminating any samples which did not have 4 image files. The test statistics might reflect that the 12 time step version is more able to adapt to missing values.
Here is the current val loss and an example plot:
<img width="504" height="468" alt="Image" src="https://github.com/user-attachments/assets/5f06deef-ca9c-41f8-a6ec-6fcb37efb389" />
<img width="1042" height="394" alt="Image" src="https://github.com/user-attachments/assets/c21a3c0d-d1a7-4a20-bdb4-2040f2d36965" />
